docs/content/dagster-cloud/account/managing-users/managing-user-roles-permissions.mdx

```diff
@@ -0,0 +1,748 @@
+      <th width="10%; fixed">Editor</th>
+      <th width="10%; fixed">Admin</th>
+      <th width="10%; fixed">
+        Organization <br />
+        Admin
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        View and create own{" "}
+        <a href="/dagster-cloud/account/managing-user-agent-tokens#managing-user-tokens">
+          user tokens
+        </a>
+      </td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+    </tr>
+    <tr>
+      <td>List all user tokens</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+    </tr>
+    <tr>
+      <td>Revoke all user tokens</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+    </tr>
+  </tbody>
+</table>
+
+### Users
+
+User management is accessed in the UI by navigating to **user menu (your icon) > Cloud Settings > Users**.
+
+<table>
+  <thead>
+    <tr>
+      <th width="50%; fixed">&nbsp;</th>
+      <th width="10%; fixed">Viewer</th>
+      <th width="10%; fixed">Launcher</th>
+      <th width="10%; fixed">Editor</th>
+      <th width="10%; fixed">Admin</th>
+      <th width="10%; fixed">
+        Organization <br />
+        Admin
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <a href="/dagster-cloud/account/managing-users">View users</a>
+      </td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+    </tr>
+    <tr>
+      <td>Add users</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+    </tr>
+    <tr>
+      <td>Edit user roles</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
+    </tr>
+    <tr>
+      <td>Remove users</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
+    </tr>
+  </tbody>
+</table>
+
+### Teams
+
+Team management is accessed in the UI by navigating to **user menu (your icon) > Cloud Settings > Teams**.
+
+**Note**: Admin users can modify teams only in deployments where they're an Admin.
```

Comment:
Only org admins can modify teams, admin users can't modify them at all (they're not deployment-scoped)

Reply:
Weird - when I tested this on Thursday I was able to edit teams as an Admin, but it seems to be as you say now. 🤷‍♀️ I'll update this!